tests/e2e/tests/login/Login.spec.ts

```diff
@@ -19,4 +19,9 @@ suite('Login test', async () => {
         await driverHelper.navigateToUrl(TestConstants.TS_SELENIUM_BASE_URL);
         await loginPage.login();
     });
+    if (TestConstants.TS_SELENIUM_LAUNCH_FULLSCREEN){
+        test('Maximize windows', async() => {
```

Comment:
I don't think that maximizing windows is something we want to test - I would include it in the Login test itself.

Reply:
Fixed